google/async_resumable_media/requests/download.py

```diff
@@ -465,18 +431,19 @@ def _add_decoder(response_raw, checksum):
         patched, or a ``_DoNothingHash`` if the decoder is patched, since the
         caller will no longer need to hash to decoded bytes.
     """
+
     encoding = response_raw.headers.get(u"content-encoding", u"").lower()
     if encoding != u"gzip":
         return checksum
 
     response_raw._decoder = _GzipDecoder(checksum)
-    return _helpers._DoNothingHash()
+    return _helpers._DoNothingHash
```

Comment:
This has changed from returning an instance to returning the entire class. Does this work? Why? If not, are we missing something in our tests?

Reply:
Looking over this it seems the error may have gotten carried forward into the tests. Digging in a bit, but I expect this will be reverted.

Reply:
Thanks